src/test/java/com/thirdcc/webapp/web/rest/EventChecklistResourceIT.java

```diff
@@ -99,52 +117,79 @@ public void setup() {
      * This is a static method, as tests for other entities might also need it,
      * if they test an entity which requires the current entity.
      */
-    public static EventChecklist createEntity(EntityManager em) {
-        EventChecklist checklist = new EventChecklist()
+    public static EventChecklist createEventChecklistEntity(EntityManager em) {
+        return new EventChecklist()
             .eventId(DEFAULT_EVENT_ID)
             .name(DEFAULT_NAME)
             .description(DEFAULT_DESCRIPTION)
             .status(DEFAULT_STATUS)
             .type(DEFAULT_TYPE);
-        return checklist;
     }
     /**
      * Create an updated entity for this test.
      *
      * This is a static method, as tests for other entities might also need it,
      * if they test an entity which requires the current entity.
      */
-    public static EventChecklist createUpdatedEntity(EntityManager em) {
-        EventChecklist checklist = new EventChecklist()
+    public static EventChecklist createUpdatedEventChecklistEntity(EntityManager em) {
+        return new EventChecklist()
             .eventId(UPDATED_EVENT_ID)
             .name(UPDATED_NAME)
             .description(UPDATED_DESCRIPTION)
             .status(UPDATED_STATUS)
             .type(UPDATED_TYPE);
-        return checklist;
+    }
+
+    public static Event createEventEntity() {
+        return new Event()
+            .name(DEFAULT_NAME)
+            .description(DEFAULT_DESCRIPTION)
+            .remarks(DEFAULT_EVENT_REMARKS)
+            .startDate(DEFAULT_EVENT_START_DATE)
+            .endDate(DEFAULT_EVENT_END_DATE)
+            .status(DEFAULT_EVENT_STATUS)
+            .venue(DEFAULT_EVENT_VENUE)
+            .fee(DEFAULT_EVENT_FEE);
     }
 
     @BeforeEach
     public void initTest() {
-        checklist = createEntity(em);
+        event = createEventEntity();
+        eventChecklist = createEventChecklistEntity(em);
+    }
+
+    @AfterEach
+    public void cleanUp() {
+        eventRepository.deleteAll();
+        eventChecklistRepository.deleteAll();
+    }
+
+    private void initEventDB() {
+        eventRepository.saveAndFlush(event);
+    }
+
+    private void initEventChecklistDB() {
+        eventChecklistRepository.saveAndFlush(eventChecklist);
     }
 
     @Test
     @Transactional
```

Comment:
I would suggest we remove all `@Transactional` annotation since we already use
```
 @AfterEach
    public void cleanUp() {
        eventRepository.deleteAll();
        eventChecklistRepository.deleteAll();
    }
```

for more information on why `@Transaction` is not encouraged, can refer to https://dev.to/henrykeys/don-t-use-transactional-in-tests-40eb

Reply:
I had remove in commit below
i do research on the @Transactional as well
just forget to remove during implementation
5cfe9941b6fc700d9145c7611ea9c9d4e19d8378

Reply:
@kachun333 Feel free to revisit the 2 new commit and resolve this conversation